### DISCUSSION TITLE
Dark mode as default?

### DISCUSSION THREAD

**Discussion by Plaenkler** [openaudible/openaudible#854] [Ideas]
Developers should not follow every trend... However, I like to leave the bright white user interfaces behind. For this reason, I would recommend using the dark mode as a default - it really enhances the look in my opinion.

**Comment by openaudible** [comment]
We use the "system" default.... whatever that is which means we don't set any foreground/background colors. 
Which is supposed to mean if your system is set to dark mode, OA should start that way. 

However, in practice, you probably need to manually set the mode to either dark, light, or system. 

We had some thoughts for future changes where a user could change any of the colors.. (light blue on dark red) but trying to keep things simple. Glad you like the dark.  :) But we're going to let the user change the preference to whatever they like. I admit I haven't fallen in love with dark mode yet.
